File: src/main/java/com/dandb/dto/verified/Phone.php
<?php

namespace models\json;

class Phone extends JSONObject{

    /** @var VerifiedValue */
    public $fax;
    /** @var VerifiedValue */
    public $toll_free_number;
    /** @var VerifiedValue */
    public $local_phone;

    function __construct()
    {
        $this->fax = new VerifiedValue();
        $this->local_phone = new VerifiedValue();
        $this->toll_free_number = new VerifiedValue();
    }


}

File: src/main/java/com/dandb/dto/verified/Phone.java
package com.dandb.dto.verified;

public class Phone {

	public VerifiedValue fax;
	public VerifiedValue toll_free_number;
	public VerifiedValue local_phone;
	public static Phone init() {
		Phone phone = new Phone();
		phone.local_phone = VerifiedValue.init();
		return phone;
	}

}